sci-mathematics/glucas/glucas-2.9.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A client to make primality test for Mersenne numbers"
+DESCRIPTION="Glucas is related to GIMPS effort to search the biggest primes \
+ever found."
+HOMEPAGE="https://sourceforge.net/projects/glucas/"
+COPYRIGHT="2001 gbvalor"
```

Comment:
Version 2.9.0 was released in 2002 so I guess we should make this tiny change:
```diff
-COPYRIGHT="2001 gbvalor"
+COPYRIGHT="2001-2002 gbvalor"
```

Reply:
How do I type in both the author's names?

Reply:
@nishanth1232: For this PR you did not use a [feature-branches](https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches).

Would you like to update the `COPYRIGHT` variable using a new PR based on a feature branch?
(There is no urgency. You can read about feature branches and open the PR when you feel you are comfortable with that topic.)

BTW, the changes would be:
```diff
-COPYRIGHT="2001-2002 gbvalor"
+COPYRIGHT="2000-2002 Guillermo Ballester Valor
+	2000-2002 Klaus Kastens"
```
and
```diff
-REVISION="1"
+REVISION="2"
```

Reply:
@fbrosson I cannot really understand about feature branches. Can I just open another pull request?

Reply:
You can, but it would be much better if you use a feature branch.
First you need to sync your https://github.com/nishanth1232/haikuports with https://github.com/haikuports/haikuports
Then you would need to create a local branch, make the changes in there.
Then you push your local branch to your remote repo.
Finally, you create a PR based on that branch.
That's not very hard. Doing it once will be a very good exercice.

I can guide you with the commands if you wish.

Reply:
https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows